launch_chffrplus.sh

```diff
@@ -101,7 +101,6 @@ function tici_init {
   sudo su -c 'echo "performance" > /sys/class/devfreq/soc:qcom,memlat-cpu0/governor'
   sudo su -c 'echo "performance" > /sys/class/devfreq/soc:qcom,memlat-cpu4/governor'
   nmcli connection modify --temporary lte gsm.auto-config yes
-  nmcli connection modify --temporary lte gsm.home-only yes
```

Comment:
Should we keep this around, in case `ui` ever fails to start and we still don't want roaming?

Reply:
Probably nice. Outside of the US roaming can cost quite a bit of money, so probably better to err on the side of caution.